README.md

```diff
@@ -235,4 +235,9 @@ $ ./run-test-docker-image.sh
 
 # You can also run the test but skip rebuilding the image 
 $ ./run-test-docker-image.sh -s
-```
\ No newline at end of file
+```
+
+Note: If you want to use a different image name and/or tag, you must ensure that the image name in docker-compose.yml matches
+the image name given to the integration test script. For example, if you run the integration tests with the following options 
+```  ./run-test-docker-image.sh -g test -t integ ```,
+then the 'image' key in docker-compose.yml must be set to 'test/volttron:integ'.
```

Comment:
You could set env variable in the docker-compose file so that the MUST goes out of this...image ${GROUP}:${TAG} or whatever